src/components/remote_control/test/smartDeviceLink.ini

```diff
@@ -1,5 +1,4 @@
 [Remote Control]
```

Comment:
@LitvinenkoIra seems you could delete whole ini-file as all other params in it is not used